RecoTauTag/Configuration/python/RecoTauTag_EventContent_cff.py

```diff
@@ -112,7 +113,9 @@
         'keep *_hpsPFTauMVA3IsolationChargedIsoPtSum_*_*',
         'keep *_hpsPFTauMVA3IsolationNeutralIsoPtSum_*_*',
         'keep *_hpsPFTauMVA3IsolationPUcorrPtSum_*_*',
-        'keep *_hpsPFTauTransverseImpactParameters_*_*'
+        'keep *_hpsPFTauTransverseImpactParameters_*_*',
```

Comment:
I think that these "keep" statements should be removed from 76X.
We are not going to make changes to the "pre-miniAOD" data tiers.